APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCChangePasscodeViewController.h

```diff
@@ -32,12 +32,15 @@
 // 
  
 #import <UIKit/UIKit.h>
-#import "APCPasscodeView.h"
+
+@class APCPasscodeView;
+
 
 @interface APCChangePasscodeViewController : UIViewController
 
-@property (weak, nonatomic) IBOutlet UILabel *textLabel;
-@property (weak, nonatomic) IBOutlet APCPasscodeView *passcodeView;
+@property (weak, nonatomic, nullable) IBOutlet UILabel *textLabel;
```

Comment:
If you're marking classes nullable you should also mark the header with NS_ASSUMES_NONNULL_BEGIN/END